src/actions/guides/authentication/browser_authentication.cr

```diff
@@ -0,0 +1,251 @@
+class Guides::Authentication::Browser < GuideAction
+  ANCHOR_ALLOW_GUESTS = "perma-allow-guests"
+  guide_route "/authentication/browser"
+
+  def self.title
+    "Browser Authentication"
+  end
+
+  def markdown
+    <<-MD
+    ## Overview
+
+    When a full Lucky app is generated with auth you will get:
+
+    * Sign up
+    * Sign in
+    * Sign out
+    * Password reset
+    * Basic profile page
+
+    ## Requiring sign in
+
+    By default actions that inherit from `BrowserAction`
+    (`src/actions/browser_action.cr`) will require sign in because
+    `BrowserAction` includes the `Auth::RequireSignIn` module
+    (`src/actions/mixins/auth/require_sign_in.cr`). This is a great way to
+    make sure that your actions are protected by default.
+
+    Typically pages that require sign in will use pages that inherit from the
```

Comment:
This reads a little weird since all pages inherit from MainLayout anyway. Not sure of a better wording at the moment though. I'll come back to this if my brain wakes up lol

Reply:
Yeah this is  confusing. I'll try rewording it